### PR TITLE
documentation 802.1x + VOIP on AlliedTelesis AT8000GS

### PR DESCRIPTION
**Pull request by lzammit**
# Description

Documentation for the VOIP with 8021.x support on AlliedTelesis AT8000GS
# Delete branch after merge

YES

### CODE REVIEW

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 1]
```diff
@@ -608,6 +607,44 @@ Finally, enable the necessary 802.1X settings:
    dot1x radius-attributes vlan
    dot1x port-control auto
 
+802.1X + VOIP
++++++++++++++
+
+
+First, activate 802.1X globally:
```
Comment (julsemaan) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Activate 802.1x:

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 2]
```diff
@@ -608,6 +607,44 @@ Finally, enable the necessary 802.1X settings:
...
+
+   dot1x system-auth-control
+
+Next, configure the RADIUS server and AAA settings:
```
Comment (julsemaan) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: RADIUS server configuration:

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 3]
```diff
@@ -608,6 +607,44 @@ Finally, enable the necessary 802.1X settings:
...
+
+   radius-server host 10.0.0.100
+   radius-server key qwerty
+   radius-server source-ip 10.0.0.14
+   aaa authentication dot1x default radius
+   aaa accounting dot1x radius
+
+Then, the LLDP configuration:
```
Comment (julsemaan) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: LLDP configuration:

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 4]
```diff
@@ -608,6 +607,44 @@ Finally, enable the necessary 802.1X settings:
...
+
+   hostname switch-name
+   ip domain-name domain.local
+   lldp med network-policy 1 voice vlan 100 vlan-type tagged dscp 34
+   lldp med network-policy 2 voice-signaling vlan 100 vlan-type tagged dscp 34
+
+Finally, enable the necessary 802.1X and VOIP settings:
```
Comment (julsemaan) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: On each interface:

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 5]
```diff
@@ -608,6 +607,44 @@ Finally, enable the necessary 802.1X settings:
...
+
+   interface ethernet g1
+   dot1x port-control force-authorized
```
Comment (julsemaan) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: indent that line and the ones below by 1 space so its clear its under the interface

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc (hunk 1 above)
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Noooooooo!

Enable 802.1X:

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 6]
```diff
@@ -608,6 +607,44 @@ Finally, enable the necessary 802.1X settings:
...
+   no dot1x radius-attributes vlan
+   no dot1x re-authentication
+   switchport mode trunk
+   switchport trunk native vlan 5
+   switchport trunk allowed vlan add 100
+   lldp med enable network-policy
+   lldp med network-policy add 1
+   lldp med network-policy add 2
+
 Amer
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc:657]: Ta mère?

Comment (julsemaan) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc:657]: https://en.wikipedia.org/wiki/Tamehr

Comment (julsemaan) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc:657]: 57 people, 24 families

Almost as big as ZammitCorp town

Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc:657]: It obviously makes senses - this is where the ZammitCorp headquarters are located.

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 7]
```diff
@@ -608,6 +607,44 @@ Finally, enable the necessary 802.1X settings:
...
++++++++++++++
+
+
+First, enable 802.1X globally:
+
+   dot1x system-auth-control
+
+Next, configure the RADIUS server configuration and AAA settings:
+
+   radius-server host 10.0.0.100
```
Comment (dwlfrth) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: 192.168.1.5 ? as per our assumptions

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 8]
```diff
@@ -608,6 +607,44 @@ Finally, enable the necessary 802.1X settings:
...
+   radius-server key qwerty
+   radius-server source-ip 10.0.0.14
```
Comment (dwlfrth) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc:631]: Should probably define on top of that section what is that IP

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 9]
```diff
@@ -608,6 +607,44 @@ Finally, enable the necessary 802.1X settings:
...
+    no dot1x radius-attributes vlan
+    no dot1x re-authentication
+    switchport mode trunk
+    switchport trunk native vlan 5
+    switchport trunk allowed vlan add 100
+    lldp med enable network-policy
+    lldp med network-policy add 1
+    lldp med network-policy add 2
+
 Amer
```
Comment (dwlfrth) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc:657]: image

Comment (julsemaan) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc:657]: image

**Pull request merged**